### PR TITLE
Upgrade the packages

### PR DESCRIPTION
**Pull request by rohansaroha**
### Ticket Link
---------------------------------------------------

### Related Links
---------------------------------------------------

### Description
---------------------------------------------------

### Steps to Reproduce / Test
---------------------------------------------------

### GIF's
---------------------------------------------------

### CODE REVIEW

File: .storybook/config.js [hunk 1]
```diff
@@ -8,20 +8,16 @@ import StoryRouter from 'storybook-router';
...
 // import enLocaleData from 'react-intl/locale-data/en';
-import {
-  translationMessages,
-  appLocales,
-  DEFAULT_LOCALE
-} from '../app/i18n.js';
+import { translationMessages, appLocales, DEFAULT_LOCALE } from '../app/i18n.js';
 
-Object.values = obj => Object.keys(obj).map(key => obj[key]);
+Object.values = (obj) => Object.keys(obj).map((key) => obj[key]);
```
Comment (rameez-wed) [inline .storybook/config.js:13]: remove brackets just `obj` should be fine. Same thing for keys

File: .storybook/config.js [hunk 2]
```diff
@@ -8,20 +8,16 @@ import StoryRouter from 'storybook-router';
...
 
 addDecorator(withSmartKnobs);
 addDecorator(withKnobs);
 
 addDecorator(StoryRouter());
 
-const getMessages = locale => translationMessages[locale];
+const getMessages = (locale) => translationMessages[locale];
```
Comment (rameez-wed) [inline .storybook/config.js:20]: remove brackets

File: babel.config.js [hunk 3]
```diff
@@ -26,7 +26,7 @@ module.exports = {
         'transform-react-remove-prop-types',
         '@babel/plugin-transform-react-inline-elements',
         '@babel/plugin-transform-react-constant-elements',
-        ['import', { libraryName: 'antd', libraryDirectory: 'es', style: true }, 'import-antd']
+        ['import', { libraryName: 'antd', /*  libraryDirectory: 'es', */ style: true }, 'import-antd']
```
Comment (rameez-wed) [inline babel.config.js:29]: remove if not required

File: internals/webpack/webpack.config.prod.js [hunk 4]
```diff
@@ -73,8 +69,8 @@ module.exports = require('./webpack.base.babel')({
         removeStyleLinkTypeAttributes: true,
         keepClosingSlash: true,
         minifyJS: true,
-        minifyCSS: true,
-        minifyURLs: true
+        minifyCSS: true
+        // minifyURLs: true
```
Comment (rameez-wed) [inline internals/webpack/webpack.config.prod.js:73]: why is this commented?

**Review by rameez-wed — CHANGES_REQUESTED** [review]

**Review by christin-wednesday — APPROVED** [review]

File: app/containers/HomeContainer/index.js [hunk 5]
```diff
@@ -51,7 +51,6 @@ export function HomeContainer({
   maxwidth,
   padding
 }) {
-  useInjectSaga({ key: 'homeContainer', saga });
   const [loading, setLoading] = useState(false);
 
   useEffect(() => {
```
Comment (rameez-wed) [inline app/containers/HomeContainer/index.js:56]: remove if unused

File: .eslintignore.js [hunk 6]
```diff
@@ -1,8 +1,5 @@
...
-stats.json
-
-.DS_Store
-npm-debug.log
-.idea
-coverage
\ No newline at end of file
+build;
+node_modules;
+stats.json.DS_Store;
```
Comment (rameez-wed) [inline .eslintignore.js:3]: .DS_Store is a different file.

File: .eslintignore.js [hunk 7]
```diff
@@ -1,8 +1,5 @@
...
+npm - debug.log.idea;
```
Comment (rameez-wed) [inline .eslintignore.js:4]: same .idea is a different file.

File: server/middlewares/frontendMiddleware.js [hunk 8]
```diff
@@ -4,16 +4,16 @@
  * Front-end middleware
  */
 module.exports = (app, options) => {
-  const isProd = process.env.NODE_ENV === 'production'
+  const isProd = process.env.NODE_ENV === 'production';
```
Comment (rameez-wed) [inline server/middlewares/frontendMiddleware.js:7]: semicolons are added everywhere can you remove those?

**Review by rameez-wed — CHANGES_REQUESTED** [review]

**Review by rameez-wed — APPROVED** [review]

File: server/middlewares/addDevMiddlewares.js [hunk 9]
```diff
@@ -1,33 +1,36 @@
...
 
-  app.use(middleware);
-  app.use(webpackHotMiddleware(compiler));
+  app.use(middleware)
+  app.use(webpackHotMiddleware(compiler))
 
   // Since webpackDevMiddleware uses memory-fs internally to store build
   // artifacts, we use it instead
-  const fs = middleware.fileSystem;
+  const fs = middleware.fileSystem
```
Comment (christin-wednesday) [inline server/middlewares/addDevMiddlewares.js:25]:
```suggestion
   const fs = middleware.context.compiler.outputFileSystem;
```

**Review by christin-wednesday — CHANGES_REQUESTED** [review]

**Review by christin-wednesday — APPROVED** [review]

**Pull request merged**